.github/workflows/make_release.yml

```diff
@@ -0,0 +1,231 @@
+          CIBW_BUILD_VERBOSITY: 2
+          # CIBW_BEFORE_BUILD: pip install certifi numpy==1.16
+        if: >
+          startsWith(github.ref, 'refs/heads/v0.17') ||
+          startsWith(github.ref, 'refs/tags/v0.17')
+      - uses: actions/upload-artifact@v2
+        with:
+          name: wheels
+          path: ./dist/*.whl
+
+  build_windows_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    if: github.repository_owner == 'viniciusdc'
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [windows-latest]
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.7'
+
+      - name: Install cibuildwheel
+        run: |
+          python -m pip install cibuildwheel
+      - name: Build wheels for CPython 3.9 (Linux and Windows)
+        run: |
+          python -m cibuildwheel --output-dir dist
+        env:
+          CIBW_BUILD: "cp39-*"
+          CIBW_MANYLINUX_X86_64_IMAGE: manylinux1
+          CIBW_MANYLINUX_I686_IMAGE: manylinux1
+          # CIBW_BEFORE_BUILD: pip install certifi numpy==1.19.3
+
+      - name: Build Windows wheels for CPython
+        run: |
+          python -m cibuildwheel --output-dir dist
+        env:
+          CIBW_BUILD: "cp3?-*"
+          CIBW_SKIP: "cp35-* cp36-* cp39-*"
+          CIBW_MANYLINUX_X86_64_IMAGE: manylinux1
+          CIBW_MANYLINUX_I686_IMAGE: manylinux1
+          # CIBW_BEFORE_BUILD: pip install certifi numpy==1.16
+
+      - name: Build wheels for CPython 3.6
+        run: |
+          python -m cibuildwheel --output-dir dist
+        env:
+          CIBW_BUILD: "cp36-*"
+          CIBW_MANYLINUX_X86_64_IMAGE: manylinux1
+          CIBW_MANYLINUX_I686_IMAGE: manylinux1
+          CIBW_BUILD_VERBOSITY: 2
+          # CIBW_BEFORE_BUILD: pip install certifi numpy==1.16
+        if: >
+          startsWith(github.ref, 'refs/heads/v0.17') ||
+          startsWith(github.ref, 'refs/tags/v0.17')
+      - uses: actions/upload-artifact@v2
+        with:
+          name: wheels
+          path: ./dist/*.whl
+
+
+  deploy:
+    name: Release
+    needs: [build_linux_37_and_above_wheels, build_macos_wheels, build_windows_wheels]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.7'
+      
+      - name: Install Dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install twine
+          pip install beautifulsoup4
+          pip install 'numpy>=1.16.5,!=1.18.0'
+      
+      - uses: actions/download-artifact@v2
+        id: download
+        with:
+          name: wheels
+          path: ./dist
+    
+      - name: Publish the source distribution on PyPI
+        if: success()
```

Comment:
imho we should push with/without success. We can manage failing tests by releasing rcs before the real thing. We've often had sporadic CI failures that we know don't actually affect the library by and large, and we want to release anyway. So it's great to *know* whether there are failing tests, but it should not stop the upload. imho! @scikit-image/core?

Reply:
I agree that we shouldn't stop releases on failure. you summarized it well.